wombase_backend/server/apps/inventory/views.py

```diff
@@ -1,3 +1,37 @@
-from django.shortcuts import render
+from rest_framework import generics, pagination
+from .models import Tool, ToolCategory
+from .serializers import ToolSerializer, ToolCategorySerializer
 
-# Create your views here.
+query_params = {
+    'cat': 'category',
+    'name': 'name',
+    'own': 'owner'
+}
+
+
+class ToolListCreateAPIView(generics.ListCreateAPIView):
+    serializer_class = ToolSerializer
+
+    def get_queryset(self):
+        queryset = Tool.objects.all()
+        for query_param, param in query_params.items():
+            if (value := self.request.query_params.get(*{query_param})) is not None:
+                queryset = queryset.filter(
+                     **{param: value} if param != "owner" else {"owner": f"+{value.strip()}"}
```

Comment:
no need to make list request by query parameters, if you can directly access employee.tool_set (or something like this) if needed